src/openrct2/world/Map.h

```diff
@@ -176,6 +176,7 @@ TileElement* tile_element_insert(const TileCoordsXYZ& loc, int32_t occupiedQuadr
 
 using CLEAR_FUNC = int32_t (*)(TileElement** tile_element, int32_t x, int32_t y, uint8_t flags, money32* price);
 
+int32_t map_place_clear_func(TileElement** tile_element, int32_t x, int32_t y, uint8_t flags, money32* price, bool is_scenery);
```

Comment:
Why was this added to the header? It will never be used.

Reply:
You're correct, it was not needed, I removed it.